src/Logging.XUnit/XUnitLogger.cs

```diff
@@ -152,21 +128,14 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState? state, Excep
         }
 
         /// <summary>
-        /// Writes a message to the <see cref="ITestOutputHelper"/> associated with the instance.
+        /// Writes a message to the <see cref="ITestOutputHelper"/> or <see cref="IMessageSink"/> associated with the instance.
         /// </summary>
         /// <param name="logLevel">The message to write will be written on this level.</param>
         /// <param name="eventId">The Id of the event.</param>
         /// <param name="message">The message to write.</param>
         /// <param name="exception">The exception related to this message.</param>
         public virtual void WriteMessage(LogLevel logLevel, int eventId, string? message, Exception? exception)
         {
-            ITestOutputHelper? outputHelper = _accessor.OutputHelper;
```

Comment:
Can we still bail out earlier if the output helper and the sink are both null?

Reply:
Yep, addressed in 0ea106f403fe20aa5d2bc5c0e1cb9afb82453714.